requirements.txt

```diff
@@ -29,7 +29,7 @@ escapism==1.*
 # Used for GCP billing management
 pandas
 # FIXME: Unpin when https://github.com/unionai-oss/pandera/issues/1656 gets solved
```

Comment:
```suggestion
```

The fixme note sais "Unpin when https://github.com/unionai-oss/pandera/issues/1656" gets solved. This is now solved and pandera 0.20 includes the fix. Due to that, I think we can go for this bump of numpy 1 to numpy 2 - but its probably fine to keep it pinned to a major version.